lib/models/messages/auxiliary/realtimecash.dart

```diff
@@ -0,0 +1,25 @@
+import 'package:rtchat/models/messages/message.dart';
+
+class SimpleRealTimeCashDonationEventModel extends MessageModel {
+  final String assetName;
```

Comment:
Make this `final AssetImage asset;` and pass the `AssetImage` into the constructor instead of the string. This keeps the resolution logic in the model.